eyeshade/controllers/accounts.js

```diff
@@ -96,9 +101,9 @@ ORDER BY created_at
       amount: Joi.number().required().description('amount in BAT'),
       settlement_currency: braveJoi.string().anycurrencyCode().optional().description('the fiat of the settlement'),
       settlement_amount: Joi.number().optional().description('amount in settlement_currency'),
-      settlement_destination_type: Joi.string().optional().valid(['uphold']).description('type of address settlement was paid to'),
+      settlement_destination_type: Joi.string().optional().valid(settlementDestinationTypes).description('type of address settlement was paid to'),
```

Comment:
I'm a bit confused about the intended use of this API given these changes.

this was originally intended to be scoped to settlement type transactions, adding the various on-chain accounts as potential settlement destination types currently doesn't make sense

also using this to display user deposits doesn't seem to fit since the destination will always be the user card, not an on-chain address

Reply:
ah. gotcha. looks like i was getting ahead of myself then. so just the transaction types below then